cmd/controller/server.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"context"
+    "net/http"
+    "time"
+
+	"github.com/golang/glog"
+	"github.com/gorilla/mux"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type prometheusMetricsServer struct {
+    http.Server
+}
+
+func prometheusHandler() http.Handler {
+	return prometheus.Handler()
+}
+
+func newPrometheusMetricsServer() *prometheusMetricsServer {
+
+	r := mux.NewRouter()
+	r.Handle("/metrics", prometheusHandler())
+
+    // Create server and register prometheus metrics handler
+    s := &prometheusMetricsServer{
+        Server: http.Server{
+            Addr:         ":9402",
```

Comment:
Can you pull this field out to a `const` at the top of the file? In future we can make this a CLI flag.